### DISCUSSION TITLE
Some new ideas for character generation.

### DISCUSSION THREAD

**Discussion by wireboy5** [Slow-Scallion4183/Adventure-Colussus#24] [Ideas]
For character generation, have you ever considered a system like dungeons and dragons? Just a thought.

**Comment by ghost** [comment]
huh? spawnpoint?

**Reply by Slow-Scallion4183** [reply]
?

**Comment by Slow-Scallion4183** [comment]
Yes i have contemplated that! It is a good idea but maybe not completely implementing dungeon and dragons just parts of it if you see what i mean :D

**Reply by wireboy5** [reply]
I see what you mean :)

**Reply by Slow-Scallion4183** [reply]
cool :D
